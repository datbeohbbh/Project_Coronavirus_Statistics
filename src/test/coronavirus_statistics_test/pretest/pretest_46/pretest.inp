50 100
? 5
q 1 43 212
q 1 45 207
q 1 30 127
q 1 31 96
q 1 42 109
q 1 46 364
q 1 47 489
q 1 29 186
q 1 36 336
q 1 26 426
? 4 8
q 1 44 398
q 3 47 377
? 4 10
? 4 5
q 1 5 194
? 4 9
q 2 36 331
q 2 31 17
? 6
? 7
q 1 5 393
q 1 15 137
? 7
? 4 10
q 1 32 197
? 7
q 1 50 382
? 4 3
q 2 26 299
? 6
q 3 15 63
? 4 15
? 7
? 6
? 6
? 4 15
q 1 13 177
? 4 1
q 3 36 1
? 6
q 1 13 236
? 4 8
? 7
q 1 1 207
q 3 43 185
q 3 29 35
q 1 43 66
q 1 48 157
q 1 10 374
q 2 31 34
? 4 18
q 1 3 396
? 6
q 3 5 84
q 1 41 79
q 1 20 380
q 3 29 34
q 3 47 73
? 4 9
q 1 28 33
? 6
q 1 37 363
q 2 26 2
? 7
q 2 10 297
q 1 38 431
q 2 29 68
? 4 4
? 4 7
? 4 20
q 2 30 46
? 4 20
? 4 5
q 2 47 29
? 7
q 3 1 166
q 3 36 1
? 6
q 3 5 179
q 1 49 434
? 4 6
? 4 10
? 6
? 4 24
? 7
? 7
? 6
? 4 14
? 6
? 4 10
? 4 22
? 4 1
? 4 7
? 4 14
? 6
? 6
? 7
? 7