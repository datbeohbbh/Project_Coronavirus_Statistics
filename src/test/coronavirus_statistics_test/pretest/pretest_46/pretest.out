Query ? 5 :
The number of safe cities is 50

Query ? 4 8 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 364
-> Has recovered : 0
-> Was dead : 0

Query ? 4 10 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 398
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 186
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 336
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 377 people

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 377 people

Query ? 4 10 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 364
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 377 people

Query ? 4 3 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 109
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 4 15 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 587
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 377 people

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 4 15 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 587
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 5
-> Has recovered : 331
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 4 8 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 186
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 377 people

Query ? 4 18 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 413
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 4 9 :
The answer is the city with the index 26 
- Information of the city 26 -
-> Number of current patients : 127
-> Has recovered : 299
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 450 people

Query ? 4 4 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 45
-> Has recovered : 51
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 10 
- Information of the city 10 -
-> Number of current patients : 77
-> Has recovered : 297
-> Was dead : 0

Query ? 4 20 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 382
-> Has recovered : 0
-> Was dead : 0

Query ? 4 20 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 382
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 49
-> Has recovered : 68
-> Was dead : 69

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 450 people

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 4 6 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 49
-> Has recovered : 68
-> Was dead : 69

Query ? 4 10 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 81
-> Has recovered : 46
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 4 24 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 413
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 450 people

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 450 people

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 4 14 :
The answer is the city with the index 48 
- Information of the city 48 -
-> Number of current patients : 157
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 4 10 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 81
-> Has recovered : 46
-> Was dead : 0

Query ? 4 22 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 396
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 3
-> Has recovered : 331
-> Was dead : 2

Query ? 4 7 :
The answer is the city with the index 15 
- Information of the city 15 -
-> Number of current patients : 74
-> Has recovered : 0
-> Was dead : 63

Query ? 4 14 :
The answer is the city with the index 48 
- Information of the city 48 -
-> Number of current patients : 157
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 6 :
The city with the highest recovery is city with the index 36
Quantity: 331 people

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 450 people

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 450 people

